pages/platform/kubernetes-k8s/configuring-oidc-provider-config/guide.en-asia.md

```diff
@@ -0,0 +1,474 @@
+---
+title: Configuring the OIDC provider on an OVHcloud Managed Kubernetes cluster
+slug: configure-oidc-provider
+excerpt: 'Find out how to configure the OIDC provider of an OVHcloud Managed Kubernetes cluster'
+section: User guides
+---
+
+**Last updated 23nd November 2022**
+
+## Objective
+
+The OVHcloud Managed Kubernetes service provides you Kubernetes clusters without the hassle of installing or operating them. 
+
+This guide will cover the feature that allows you to secure your OVHcloud Managed Kubernetes cluster with OpenID Connect (OIDC).
+
+In this guide you will be able to configure your OIDC provider through the OVHcloud Control Panel, OVHcloud API and Terraform.
+
+## Requirements
+
+- A [Public Cloud project](https://www.ovhcloud.com/asia/public-cloud/) in your OVHcloud account
+- Access to the [OVHcloud Control Panel](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/asia/&ovhSubsidiary=asia)
+- An OVHcloud Managed Kubernetes cluster
+
+## Warning
+
+> [!warning]
+> Several elementary checks, on provider URL, CA content & required claims, for example, exists but if you misconfigured your OIDC provider, error logs will be displayed on Kubernetes API Server side.
+
+## Instructions
+
+### Configure OIDC provider through the OVHcloud Control Panel
+
+Log in to the [OVHcloud Control Panel](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/asia/&ovhSubsidiary=asia), go to the `Public Cloud`{.action} section and select the Public Cloud project concerned.
+
+Access the administration UI for your OVHcloud Managed Kubernetes clusters by clicking on `Managed Kubernetes Service`{.action} in the left-hand menu.
+
+![Clusters](images/clusters.png){.thumbnail}
+
+Click on your Kubernetes cluster.
+
+![Your Kubernetes cluster](images/cluster.png){.thumbnail}
+
+Click on the `...` button in the `OIDC provider` section.
+
+![Configure OIDC provider menu](images/configure-oidc-menu.png){.thumbnail}
+
+Fill the needed information in the popup:
+
+* `Provider URL`: The OIDC issuer url.
+* `Client ID`: The OIDC client ID.
+* `OIDC Username Claim`: JWT claim to use as the user name. By default sub, which is expected to be a unique identifier of the end user. Admins can choose other claims, such as email or name, depending on their provider. However, claims other than email will be prefixed with the issuer URL to prevent naming clashes with other plugins.
+* `OIDC Username Prefix`: Prefix prepended to username claims to prevent clashes with existing names (such as `system:users`). For example, the value `oidc:` will create usernames like `oidc:jane.doe`. If this field isn't set and `oidcUsernameClaim` is a value other than email the prefix defaults to `issuer_url` where `issuer_url` is the value of `oidcIssuerUrl.` The value - can be used to disable all prefixing.
```

Comment:
```suggestion
* `OIDC Username Prefix`: Prefix prepended to username claims to prevent clashes with existing names (such as `system:users`). For example, the value `oidc:` will create usernames like `oidc:jane.doe`. If this field isn't set and `oidcUsernameClaim` is a value other than email the prefix defaults to `issuer_url` where `issuer_url` is the value of `oidcIssuerUrl.` The value `-` can be used to disable all prefixing.
```